3498/1/lib/load_gems.rb

```diff
@@ -0,0 +1,80 @@
+# :reek:InstanceVariableAssumption
```

Comment:
You need to get rid of ALL reek and rubocop disables. Only after that I can proceed with review